index.html

```diff
@@ -22,31 +21,272 @@
+            <div class="work1-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="work-description-detail">
+                <ul>
+                    <li><a class="text-decoration1 blue-color">HTML</a></li>
+                    <li><a class="text-decoration1 blue-color">CSS</a></li>
+                    <li><a class="text-decoration1 blue-color">JavaScript</a></li>
+                </ul>
+            </div>
+            <div class="btn-see-project">
+                <a class="text-decoration blue-color">
+                    <button class="btn">See project</button>
+                </a>
+            </div>
+        </div>
+    </section>
+    <section class="about-myself">
+        <div class="about-myself-container">
+            <div class="title">
+                <h2>About Myself</h2>
+            </div>
+            <div class="description">
+                <p>Hello I'm a software developer!. I can help you build a product, feature or website. Look through
+                    some
+                    of my work and experience!. If you like what you see and have a project you need coded, don't
+                    hesitate to
+                    contact
+                    me.
+                </p>
+            </div>
+            <div class="lets-connect">
+                LET'S CONNECT
+            </div>
+            <ul class="icons-socialMedia2">
+                <li class="iconnet iconnetMar">
+                    <a href="https://github.com/pradmon"><img src="./img/github.png" alt="github-icon"></a>
+                </li>
+                <li class="iconnet iconnetMar">
+                    <a href="https://www.linkedin.com/in/jorgepradamoncada/"><img src="./img/linkedin.png"
+                            alt="linkedin-icon"></a>
+                </li>
+                <li class="iconnet iconnetMar">
+                    <a href="https://www.angellist.com/"><img src="./img/angellist.png" alt="angellist-icon"></a>
+                </li>
+                <li class="iconnet iconnetMar">
+                    <a href="https://twitter.com/praduar"><img src="./img/twitter1.png" alt="twitter-icon"></a>
+                </li>
+                <li class="iconnet iconnetMar">
+                    <a href="https://medium.com/" rel="noopener noreferrer"><img src="./img/medium.png"
+                            alt="medium-icon"></a>
+                </li>
+            </ul>
+            <div class="summary">
+                <a class="text-decoration blue-color">
+                    <button class="btnsummary btn">Get my resume</button>
+                </a>
+            </div>
+            <ul class="languageMar">
+                <li class="iconnetMar">
+                    <div class="language">
+                        <div class="language-title">
+                            <div class="title1">Languages</div>
```

Comment:
I suggest you fix this and use the semantic element list `<ul>`. As per the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), this here should be a list of lists
languages, Frameworks, and skills are lists
inside language is another list with Javascript, Html, and CSS
<img width="1672" alt="Screenshot 2023-04-29 at 08 03 47" src="https://user-images.githubusercontent.com/48546992/235284562-7ed7ca43-aa51-44d2-aa3a-6b391468f1b2.png">